src/applications/vaos/new-appointment/components/ClinicChoicePage/index.jsx

```diff
@@ -33,19 +37,13 @@ function vowelCheck(givenString) {
   return /^[aeiou]$/i.test(givenString.charAt(0));
 }
 
-function getPageTitle(schema, typeOfCare, usingPastClinics) {
-  const typeOfCareLabel = formatTypeOfCare(typeOfCare.name);
-  let pageTitle = 'Choose a VA clinic';
-  if (schema?.properties.clinicId.enum.length === 2 && usingPastClinics) {
-    pageTitle = `Make ${
-      vowelCheck(typeOfCareLabel) ? 'an' : 'a'
-    } ${typeOfCareLabel} appointment at your last clinic`;
-  }
-  return pageTitle;
-}
-
```

Comment:
Removing the `getPageTitle` function because regardless of multiple clinics or single clinic, the page title will display "Choose a VA clinic" based on [content document](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/content/schedule-va.md#choose-a-va-clinic)